frontend/control-center/src/App.tsx

```diff
@@ -6,12 +6,6 @@ import styles from './App.module.scss';
 import {getClientConfig} from './actions/config';
 import {Navigate, Route, Routes} from 'react-router-dom';
 import {INBOX_ROUTE, CATALOG_ROUTE, CONNECTORS_ROUTE, ROOT_ROUTE, STATUS_ROUTE, WEBHOOKS_ROUTE} from './routes/routes';
-import FacebookConnect from './pages/Connectors/Providers/Facebook/Messenger/FacebookConnect';
-import ChatPluginConnect from './pages/Connectors/Providers/Airy/ChatPlugin/ChatPluginConnect';
-import TwilioSmsConnect from './pages/Connectors/Providers/Twilio/SMS/TwilioSmsConnect';
-import TwilioWhatsappConnect from './pages/Connectors/Providers/Twilio/WhatsApp/TwilioWhatsappConnect';
-import GoogleConnect from './pages/Connectors/Providers/Google/GoogleConnect';
-import InstagramConnect from './pages/Connectors/Providers/Instagram/InstagramConnect';
 import NotFound from './pages/NotFound';
```

Comment:
these are unsued variables because of my previous cleanup PRs on the Control Center.